src/kOS.Safe/Compilation/Opcode.cs

```diff
@@ -1469,10 +1554,10 @@ public static object ExecuteDelegate(ICpu cpu, Delegate dlg)
                 {
                     dlg.DynamicInvoke(argArray);
                     return null; // So that the compiler building the opcodes for a function call statement doesn't
-                                 // have to know the function prototype to decide whether or
-                                 // not it needs to pop a value from the stack for the return value.  By adding this,
-                                 // it can unconditionally assume there will be exactly 1 value left behind on the stack
-                                 // regardless of what function it was that was being called.
+                    // have to know the function prototype to decide whether or
+                    // not it needs to pop a value from the stack for the return value.  By adding this,
+                    // it can unconditionally assume there will be exactly 1 value left behind on the stack
+                    // regardless of what function it was that was being called.
```

Comment:
dangit I got lazy and told the editor to auto-indent this section, and it did this to my nicely lined up comment.